src/meta/src/rpc/service/meta_member_service.rs

```diff
@@ -47,7 +47,10 @@ impl MetaMemberService for MetaMemberServiceImpl {
             Either::Left(election_client) => {
                 let mut members = vec![];
                 for member in election_client.get_members().await? {
-                    let host_addr = member.id.parse::<HostAddr>()?;
+                    let host_addr = member
```

Comment:
`host_addr = HostAddr::from_str(&member.id)?`. RwError will convert to tonic::Status

Reply:
I've changed the error type to `anyhow::Error` here (because it's really inconvenient to convert `RwError` to `MetaError`), so we have to explicitly map the error here.